### ISSUE TITLE
[Bug] Swipe to change layout messed prevents scrolling right-left

### ISSUE THREAD

**Issue by frsechet** [HappenApps/Quiver#684]
On 3.0.x, when in a diagram cell and in split mode, if the diagram is larger than the screen and you need to scroll left-right to see the content, the "swipe to change layout" takes over, and you can not scroll to see the whole content of your diagram.

**Comment by ylian** [comment]
Disable "swipe to change layout" in the preferences as a workaround.

**Comment by frsechet** [comment]
Of course, but... that's only a workaround. Expected behavior could be that scroll has priority when needed, and if you go beyond the "bounce", then only does it change layout. Something like that would make sense. :-)

**Comment by ylian** [comment]
Yes indeed. Until I figure out how to actually implement that. :)
